plugins/ipfs/local/localipfs.go

```diff
@@ -0,0 +1,504 @@
+package pluginlocalipfs
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/ipfs/go-cid"
+	config "github.com/ipfs/go-ipfs-config"
+	serial "github.com/ipfs/go-ipfs-config/serialize"
+	"github.com/ipfs/iptb/plugins/ipfs"
+	"github.com/ipfs/iptb/testbed/interfaces"
+	"github.com/ipfs/iptb/util"
+	"github.com/multiformats/go-multiaddr"
+	"github.com/pkg/errors"
+)
+
+var errTimeout = errors.New("timeout")
+
+var PluginName = "localipfs"
+
+type Localipfs struct {
```

Comment:
nit: `LocalIpfs` and `DockerIpfs` seem to be more inline with other naming I have seen, especially as they are not a single word, but rather `Local IPFS` and `Docker IPFS` if you write them not as identifiers

Reply:
Yeah, `LocalIpfs` seems better.